### ISSUE TITLE
Multiple warnings during installer build using vagrant

### ISSUE THREAD

**Issue by Hooverdan96** [rockstor/rockstor-installer#181]
During recent build tests using vagrant, these warning messages popped up:

```
[ WARNING ]: 20:19:14 | Implicitly creating root volume
[ WARNING ]: 20:19:14 | --> Future versions of kiwi will not do this anymore
[ WARNING ]: 20:19:14 | --> Please specify btrfs_root_is_subvolume true|false
```
and
```
[ WARNING ]: 20:19:41 | Theme ../kiwi-images/build/image-root/boot/grub2/themes/upstream not found
[ WARNING ]: 20:19:41 | Set bootloader terminal to console mode
[ INFO    ]: 20:19:41 | --> Using prebuilt unsigned efi image: ../kiwi-images/build/image-root/usr/share/grub2/x86_64-efi/grub.efi
```
and
```
[ INFO    ]: 20:19:41 | --> Syncing EFI boot data to EFI partition
[ WARNING ]: 20:19:41 | Extended attributes not supported for target: /var/tmp/kiwi_mount_manager.1vmn8jmj
```
This should probably be addressed at some point.

**Comment by Hooverdan96** [comment]
I assume, for the first warning the parameter `btrfs_root_is_subvolume` has to be inserted for all three architectures in this corresponding statement and be set to true:
```
<type image="oem" initrd_system="dracut" filesystem="btrfs" fsmountoptions="noatime,compress=lzo" firmware="efi" kernelcmdline="plymouth.enable=0 rd.kiwi.oem.maxdisk=5000G earlycon" bootpartition="false" devicepersistency="by-uuid" btrfs_root_is_subvolume=”true" btrfs_root_is_snapshot="true" btrfs_quota_groups="false" efipartsize="64" format="qcow2">
```
and

```
<type image="oem" initrd_system="dracut" filesystem="btrfs" fsmountoptions="noatime,compress=lzo" firmware="efi" kernelcmdline="plymouth.enable=0 rd.kiwi.oem.maxdisk=5000G earlycon" bootpartition="false" devicepersistency="by-uuid" btrfs_root_is_subvolume=”true" btrfs_root_is_snapshot="true" btrfs_quota_groups="false" efipartsize="64" format="qcow2">
```
and
```
<type image="oem" initrd_system="dracut" filesystem="btrfs" fsmountoptions="noatime,compress=lzo" firmware="efi" kernelcmdline="plymouth.enable=0 rd.kiwi.oem.maxdisk=5000G console=ttyS0,115200 console=tty" bootpartition="false" devicepersistency="by-uuid" btrfs_root_is_subvolume=”true" btrfs_root_is_snapshot="true" btrfs_quota_groups="false" efipartsize="64" editbootinstall="editbootinstall_rpi.sh">
```

**Comment by phillxnet** [comment]
@Hooverdan96 Thanks for creating this issue, it looks like we have had some overlap with @FroggyFlox original issue which I was updating yesterday while you were detailed essentially the same changes here :).

Re your: https://github.com/rockstor/rockstor-installer/issues/147#issuecomment-2489751020 there, I'll close this one as I think we have mostly duplicate info. And we also looks to have come to the same conclusion.

However great to get some vagrant testing in there. I don't see the vagrant method as a priority myself, and it depends on yet more image availability. But still, good to know it still works for some images.

Also we need to specify the version of kiwi-ng we are using as I've now moved to using 10.1.18 for the downloadable installers: at least for now.

Closing as duplicate as per @Hooverdan96 above comment.
